tests/components/auth/test_login_flow.py

```diff
@@ -66,7 +66,7 @@ async def test_invalid_username_password(hass, aiohttp_client):
             },
         )
 
-    assert resp.status == HTTPStatus.OK
+    assert resp.status == HTTPStatus.UNAUTHORIZED
     step = await resp.json()
     assert len(mock_process_wrong_login.mock_calls) == 1
```

Comment:
I expect this assertion to fail. The method is already decorated with `@log_invalid_auth` so the call to `await process_wrong_login(request)` when the status code is overridden is now logging a wrong login a 2nd time.